### ISSUE TITLE
adding new gestures

### ISSUE THREAD

**Issue by aadarshpratik1** [asingh33/CNNGestureRecognizer#14]
hi ! 
I am new to this part of coding and am unable to understand how to add a new gesture to the given setup .
could you walk me through it . ?? 
thanks

**Comment by asingh33** [comment]
Hey,
For doing that you will require to capture new sample images (option available in the script) and then retrain the CNN network along with it. 
I might be able to reply back with detailed steps by end of this week only.

**Comment by aadarshpratik1** [comment]
Ohh..... I figured it out..... But forgot to close the issue..... Sorry about that...... 
What I don't get is how does the program understand which gesture a sample image represents......does it take it from the name or smtg?? 
Anyway..... Thnxs for the program...... It works great

**Comment by aadarshpratik1** [comment]
Hey,
Sorry to bother you again, but i am a complete beginner in this field.
I was able to train new finger gestures, but the problem statement of my
project changed a little, so i was wondering whether the same program can
be retrained to work properly for complete arm gestures like recognizing
gestures like raised right arm, both arm raised, etc.
Also, i was wondering, when i added gestures, I just took images and named
them after the gesture, and trained it, but i was wondering while training
how does the program know which gesture a picture is a sample of.

Thank you again , and sorry for bothering you,
Aadarsh Pratik
ᐧ
